student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative('./person')
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, age, name, parent_permission)
+    @classroom = classroom
+    super(age, name, parent_permission)
```

Comment:
- [ ] Since name and parent_permission are optional and have default values of "Unknown" and true you have to write it while initializing:
```ruby
class Student < Person 
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

You should also put super right after initialization as shown in example.

Reply:
Thank you @ShoiraTa for reviewing. I have fixed this issue.